.claude-plugin/plugins/oh-mcp/commands/setup.md

```
@@ -0,0 +1,59 @@
+mkdir -p ~/.config/openhorizons
+```
+
+Then write to `~/.config/openhorizons/config.json`:
+```json
+{
+  "api_key": "<USER_PROVIDED_KEY>",
+  "api_url": "https://app.openhorizons.me"
+}
+```
+
+Tell them to get a key from https://app.openhorizons.me/settings/api-keys if they don't have one.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)